package.json

```diff
@@ -5,8 +5,8 @@
   "config": {
     "webservice_version": "1.15.0",
     "use_circle": true,
-    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/9967/workflows/e03b3b8b-aec8-4316-9918-995add989dd1/jobs/34030/artifacts",
-    "circle_build_id": "34030"
+    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/10157/workflows/f70aa9e7-433e-453c-a874-5a61156c25cd/jobs/35423/artifacts",
```

Comment:
This is pointing to non-develop branch artifacts. Will update when dockstore/dockstore#5590 is merged.